Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -210,6 +211,16 @@ namespace RedirectOutput
         });
 
         SetRedirection("stderr", g_redirect_stderr_saved, g_redirect_stderr, []([[maybe_unused]] const char* msg) {
+            if (lastPythonError.empty())
+            {
+                lastPythonError = msg;
+                const int lengthOfErrorPrefix = 11;
```

Comment:
I think there might be a safer way to get this length, saving the error prefix somewhere as a const and check length (in case it is changed)